filament/src/components/LightManager.cpp

```diff
@@ -482,6 +482,13 @@ float LightManager::getSpotLightOuterCone(Instance i) const noexcept {
     return upcast(this)->getSpotParams(i).outerClamped;
 }
 
+float LightManager::getSpotLightInnerConeApproximate(Instance i) const noexcept {
+    float cosOuter = fast::cos(getSpotLightOuterCone(i));
```

Comment:
use `std::cos` here -- since we're computing some value back, might as well try to be precise.

Reply:
👍